### DISCUSSION TITLE
在一些电脑，将开始菜单调到Win10之后任务栏和通知栏的反应变慢，win10开始菜单无法打开的问题

### DISCUSSION THREAD

**Discussion by ghost** [zetaloop/ExplorerPatcher#34] [Q&A]
使用机型是Matebook E和Matebook 16s。两个都已经更新了Win11（正式版）之后也装上了Explorer Patcher。但是Matebook E使用正常，16s却出现了如题目上的问题。具体是在Win10任务栏状态和Win10开始菜单下，所有固定和打开的图标需要等待1到2分钟才会出现，Wifi和通知栏打开反应慢或无反应，Win10开始菜单无法打开。调到Win11开始菜单就会恢复。Win11任务栏状态和Win10开始菜单下，出现Win11任务栏需要等待1到2分钟才会出现，通知栏打开反应慢或无反应，不过一旦成功打开就恢复了，但是Win10开始菜单依然无法打开。目前我初步判定是Win10开始菜单的相关组件出现问题所导致的。

但是E不会出现这种问题，就只有16s出现了这个问题。希望能回复一下原因和具体解决办法（除了重置以外的），谢谢作者。

**Comment by zetaloop** [comment]
是否安装了3.0修补包？若恢复文件（如何恢复：https://github.com/zetaloop/ExplorerPatcher/issues/6#issuecomment-1653758283 ）是否还有问题？

*ps: 作为中文翻译，我对ep修改系统的原理并不是完全清楚，我主要修改关于字符显示的部分。如果您对ep软件的工作产生疑问，可询问原作者valinet（问题汇报最好使用英文原版重现），不过原作者可能工作比较繁忙而处理周期较长，您也可询问原软件的社区。https://github.com/valinet/ExplorerPatcher*

**Reply by ghost** [reply]
已经解决了。可能是Windows中的一个bug。在重装之后，安装Explorer Patcher，此时开始菜单是打不开的状态。然后再更新Windows之后，Explorer Patcher就会消失，然后任务栏也会莫名重启（但是文件资源管理器可以打开，任务栏不动就不会出现问题）。此时再安装Explorer Patcher之后，Win10开始菜单就可以打开了。（某些电脑需要取消修改右键菜单）

**Reply by ghost** [reply]
谢谢建议

**Comment by ghost** [comment]
安装了3.0，但是还是出现这个问题。
关于恢复文件，C:\Windows\SystemApps\Microsoft.Windows.StartMenuExperienceHost_cw5n1h2txye是无法打开的，看一下到底是什么问题
| |
pakkinam
|
|
***@***.***
|
---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2023年7月31日 18:54 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***> ,
***@***.***> |
| 主题 | Re: [zetaloop/ExplorerPatcher] 在一些电脑，将开始菜单调到Win10之后任务栏和通知栏的反应变慢，win10开始菜单无法打开的问题 (Discussion #34) |

是否安装了3.0修补包？若恢复文件（如何恢复：#6 (comment) ）是否还有问题？

ps: 作为中文翻译，我对ep修改系统的原理并不是完全清楚，我主要修改关于字符显示的部分。如果您对ep软件的工作产生疑问，可询问原作者valinet（问题汇报最好使用英文原版重现），不过原作者可能工作比较繁忙而处理周期较长，您也可询问原软件的社区。https://github.com/valinet/ExplorerPatcher

**Reply by zetaloop** [reply]
你好，抱歉，我之前打错了，路径应该是：`%SystemRoot%\SystemApps\Microsoft.Windows.StartMenuExperienceHost_cw5n1h2txyewy`
